Sources/TransifexNative/Locales.swift

```diff
@@ -46,43 +61,53 @@ public final class LocaleState : NSObject {
     
     /// The currently selected locale that is being displayed in the app
     @objc
-    public var currentLocale : String {
+    public var currentLocale: String {
         get {
             return currentLocaleProvider.currentLocale()
         }
     }
     
     /// A list of all locales supported in the app
     @objc
-    public private(set) var appLocales : [String]
+    public private(set) var appLocales: [String]
```

Comment:
Let's be more specific in the docstring, mentioning that it also includes the source locale.

Reply:
Done!